modules/virt-preparing-container-disk-for-vms.adoc

```diff
@@ -7,6 +7,19 @@
 
 You must build a container disk with a virtual machine image and push it to a container registry before it can used with a virtual machine. You can then either import the container disk into a PVC using a data volume and attach it to a virtual machine, or you can attach the container disk directly to a virtual machine as an ephemeral `containerDisk` volume.
 
+[NOTE]
+====
+The size of a disk image inside a containerDisk is limited by the size of the maxium layer size of any registry where the containerDisk will be hosted.
+For Red Hat Quay the default limit is 20GiB and can be configured using the `MAXIMUM_LAYER_SIZE` in the https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html-single/manage_red_hat_quay/index#quay-schema[Red Hat Quay configuration].
+====
+
+[IMPORTANT]
+====
+Large container images - and thus containerDisks - can lead to disk pressure on worker nodes and in the worst case lead to node unavailability.
+NEEDS reference
```

Comment:
We could link to https://access.redhat.com/solutions/3396081

Reply:
Thanks @fabiand - Yes we can use a LINK to that KB as long as its a verified solution.